examples/hitl/rearrange/rearrange.py

```diff
@@ -433,6 +454,76 @@ def sim_update(self, dt, post_sim_update_dict):
 
         self._update_help_text()
 
-    def is_app_state_done(self):
-        # terminal neverending app state
-        return False
+
+class AppStateRearrangeTutorialTransition(AppState):
```

Comment:
This is a new AppState that essentially wraps two other AppStates (Rearrange and Tutorial) and handles the transition between them (when the tutorial ends).